LabelsCityScapesDB:
  cannyThresh: 100
  morphThresh: 5
  polyDPThesh: 3
  vegetation:
    name: "vegetation"
    R: 107
    G: 142
    B: 35
  person:
    name: "person"
    R: 220
    G: 20
    B: 60
  rider:
    name: "rider"    
    R: 255
    G: 0
    B: 0
  car:
    name: "car"    
    R: 0
    G: 0
    B: 142
  truck:
    name: "truck"    
    R: 0
    G: 0
    B: 70
  bus:
    name: "bus"    
    R: 0
    G: 60
    B: 100
  caravan:
    name: "caravan"    
    R: 0
    G: 0
    B: 90
  trailer:
    name: "trailer"    
    R: 0
    G: 0
    B: 110
  motorcycle:
    name: "motorcycle"    
    R: 0
    G: 0
    B: 230
  bicycle:
    name: "bicycle"    
    R: 119
    G: 11
    B: 32

LabelsKittyDB:
  car: "Car"
  van: "Van"
  truck: "Truck"
  pedestrian: "Pedestrian"
  person_sit: "Person_sitting"
  cyclist: "Cyclist"

# 11: img file resizer
# 22: xml file generator
# 33: xml file checker
KittyDBConverter:
  feature: 33

#"/Research_DB/workdone_etriDB/v1.0/img/"
#"/Research_DB/workdone_etriDB/v1.0/label/"
KittyDBfolder:
  image: "/Research_DB/raw_etriDB/v1.0/ETRI_kittyDB/image/" #"/kittyDBsmall/image/"
  label: "/Research_DB/raw_etriDB/v1.0/ETRI_kittyDB/label/" #"/kittyDBsmall/label/"
  cvtimg: "/Research_DB/raw_etriDB/v1.0/ETRI_kittyDB/raw_vga/" #"/kittyDBsmall/cvtimg/"
  xml_label: "/Research_DB/raw_etriDB/v1.0/ETRI_kittyDB/xml_label/" #"/kittyDBsmall/xml_label/"
  imgfile_type: "*.png"
  txtfile_type: "*.txt"
  xmlfile_type: "*.xml"
  imgfile_extension: "png"
  xmlfile_extension: "xml"  
  file_name_fwd: "etridb_ca_"
  file_name_num_digit: 6
  cvtimg_width: 640
  cvtimg_height: 480

# 1: img file resizer
# 2: xml file generator
# 3: xml file checker
CityScapesDBConverter:
  feature: 3
  use_pet_mixing: false
  trial_pet_mixing: 3000
  mix_width_ratio: 0.1
  mix_height_ratio: 0.25
  mix_inner_ratio: 0.1

CityScapesDBfolder:
  raw: "/Research_DB/raw_etriDB/v1.0/ETRI_cityscapeDB/raw/"                   #"/cityscapeDBsmall/raw/"
  cvtimg: "/Research_DB/raw_etriDB/v1.0/ETRI_cityscapeDB/raw_vga/"            #"/cityscapeDBsmall/cvtimg/"
  color_label: "/Research_DB/raw_etriDB/v1.0/ETRI_cityscapeDB/color_label/"   #"/cityscapeDBsmall/color_label/"
  polygon_data: "/Research_DB/raw_etriDB/v1.0/ETRI_cityscapeDB/polygon/"      #"/cityscapeDBsmall/polygon/"
  xml_label: "/Research_DB/raw_etriDB/v1.0/ETRI_cityscapeDB/xml_label/"       #"/cityscapeDBsmall/xml_label/"
  pet_mix: "/Research_DB/raw_etriDB/v1.0/ETRI_cityscapeDB/pet_db/"            # pet imgs
  imgfile_type: "*.png"
  polygonfile_type: "*.json"
  xmlfile_type: "*.xml"
  imgfile_extension: "png"
  xmlfile_extension: "xml"
  file_name_fwd: "etridb_ca_"
  file_name_num_digit: 6
  cvtimg_width: 640
  cvtimg_height: 480  

       